source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACOHostConfig.mm

```diff
@@ -341,4 +335,70 @@ + (NSArray *)getConstraintsForImageAlignment:(HorizontalAlignment)alignment
     return dateParsedString;
 }
 
++ (UIColor *)convertHexColorCodeToUIColor:(const std::string&)hexColorCode
+{
+    if((hexColorCode.length() < 2) || (hexColorCode.at(0) != '#') || !isxdigit(hexColorCode.at(1)) ||
+       ((hexColorCode.length() != 7) && hexColorCode.length() != 9)) {
+        NSLog(@"invalid hexcolor code is given for background color: %@",
```

Comment:
>background color [](start = 51, length = 16)

This method isn't just used for background colors.

Reply:
thanks!
you are absolutley correct.

---
In reply to: [220399342](https://github.com/Microsoft/AdaptiveCards/pull/1954#discussion_r220399342) [](ancestors = 220399342)

Reply:
You should update the log output so it doesn't say "background color".

---
In reply to: [220657960](https://github.com/Microsoft/AdaptiveCards/pull/1954#discussion_r220657960) [](ancestors = 220657960,220399342)

Reply:
I missed that one; i will make the correction. Thanks

---
In reply to: [221090575](https://github.com/Microsoft/AdaptiveCards/pull/1954#discussion_r221090575) [](ancestors = 221090575,220657960,220399342)